Adding the refactor of the Shape example

rickonnen opened the pull request.

rickonnen committed:
Adding the refactor of the Shape example

carledriss commented:
@rickonnen 
Let me know when you have fixed the checkstyle errors.

carledriss commented on src/main/java/com/jalasoft/selenium/giovanny/movies/Customer.java:
```diff
@@ -0,0 +1,65 @@
+        return _name;
+    }
+
+    public String statement() {
```
this method is doing 3 things.
Please split it.

carledriss commented:
@rickonnen 
please update against upstream/develop

Commit:
Merge remote-tracking branch 'upstream/develop' into develop

Commit:
Second commit changes

Giovanny Uscamaita

The commit changes include the two exercises about Shapes and Moviews refactoring

Note: I have a doubt about why the checkstyle had asked me to convert most method parameters to final.
